### PR TITLE
Remove legacy member UI

### PR DESCRIPTION
**Pull request by ritesh-skyach**
- Remove `yarn`;
- Remove Gems (rails-i18n, gon, angularjs-rails);
- Remove /assets for funds page;
- Remove application_controller methods related to funds page;
- Remove /private and /concern controllers;
- Remove application_helpers methods related to funds page;
- Rewrite links and notices for deposits and withdraws in admin panel;
- Remove private views and templates;
- Remove config/locales;
- Clean routes.rb and application.yml.erb;

Co-authored-by: Dinesh Chohda <dinesh.skyach@gmail.com>
Co-authored-by: Ritesh Chohda <ritesh.skyach@gmail.com>
Co-authored-by: Yaroslav Savchuk <ysavchuk@heliostech.fr>

### CODE REVIEW

**Comment by ysv** [comment]
TODO: 

- [x] Rebase on master

- [x] Check application.yml.erb

- [x] Check app/views

- [x] Remove methods from models which were used by legacy controllers

**Comment by ysv** [comment]
- [x] update Dockerfile

- [x] Do we need vendor/assets/yarn_components ?

- [x] remove raven from FE side 

- [x] application.yml.erb METADATA_\*, BANK_DETAILS_HTML, DISABLE_\*_UI

**Comment by dinesh-skyach** [comment]
@ysv Do we need to remove `* METADATA_` from application.yml.erb?

**Comment by mnaichuk** [comment]
I think you can also remove this dropdown 
deepinscreenshot_select-area_20190219154947

**Comment by ysv** [comment]
> @ysv Do we need to remove `* METADATA_` from application.yml.erb?

@dinesh-skyach Agree we still need this

**Comment by ysv** [comment]
- [x] Remove I18n stuff

- [x] lib/market_constraints.rb

- [x] https://github.com/rubykube/peatio/pull/2051#issuecomment-465136227

File: .travis.yml [hunk 1]
```diff
@@ -28,14 +28,12 @@ before_install:
   - . $HOME/.nvm/nvm.sh
```
Comment (ysv) [inline .travis.yml]: Don't need node too

**Review by ysv — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (dinesh-skyach) [reply .travis.yml]: @ysv Removed!

**Comment by shal** [comment]
Looks nice

**Review by ysv — APPROVED** [review]

**Review by mod — APPROVED** [review]

**Pull request merged**